torch/_inductor/codegen/wrapper.py

```diff
@@ -135,6 +135,9 @@ def convert_arg_type(arg: torch.Argument) -> str:
         else:
             return f"at::{python_type} const&"
 
+    if python_type == "List[Optional[Tensor]]":
```

Comment:
This `convert_arg_type` is not used anymore. Are you sure the change here is necessary?